tools/rosnode/src/rosnode/__init__.py

```diff
@@ -62,6 +63,7 @@
 
 NAME='rosnode'
 ID = '/rosnode'
+CONNECTION_PATTERN = re.compile(r'\w+ connection on port (\d+) to \[(.*) on socket (\d+)\]')
```

Comment:
It's too bad there doesn't seem to be a more robust way to get this information than tying it to the textual output from here:

https://github.com/ros/ros_comm/blob/e96c407c64e1c17b0dd2bb85b67f388380527097/clients/roscpp/src/libros/transport/transport_tcp.cpp#L719-L724